src/main/kotlin/de/craftan/engine/TurnSequence.kt

```diff
@@ -0,0 +1,24 @@
+package de.craftan.engine
+
+/**
+ * Models the sequence the players play in
+ * @param players An list ordered by the turn order
+ * @see CraftanGame
+ */
+class TurnSequence(
+    private val players: List<CraftanPlayer>,
+) {
+    private var currentPlayerIndex: Int = 0
+
+    /**
+     * Returns the next player to have his turn
+     */
+    fun getNextPlayer(): CraftanPlayer {
+        if (currentPlayerIndex < players.size - 1) {
+            currentPlayerIndex++
```

Comment:
When the index is zero, it will be incremented to 1 before returning the first player. So player 0 is skipped, until the counter is reset. So the sequence is shifted by one